daemon/info.go

```diff
@@ -63,9 +63,24 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 
 	sysInfo := sysinfo.New(true)
 
+	var cRunning, cPaused, cStopped int
```

Comment:
Getting ahead of myself since we're still in design review, but... ideally I would prefer if we just kept track of this info as each container is started/stopped/.... then we could avoid the loop.  However, that's a bigger change and since this func is only called by the API layer (meaning its not called by the engine itself for some reason) the only hit (from a performance perspective) will be to the user who called the /info API, its probably ok for now.  But, if we get into cases where there are too many containers and this func because too slow then we should reconsider how we calculate this.